tests/src/test/java/alluxio/master/file/FileSystemMasterClientRestApiTest.java

```diff
@@ -179,49 +159,21 @@ public void getStatusTest() throws Exception {
     FileInfo fileInfo = FileInfoTest.createRandom();
     Mockito.doReturn(fileInfo).when(mFileSystemMaster).getFileInfo(Mockito.<AlluxioURI>any());
 
-    TestCaseFactory
-        .newMasterTestCase(getEndpoint(FileSystemMasterClientRestServiceHandler.GET_STATUS), params,
-            "GET", fileInfo, mResource).run();
+    new TestCase(mHostname, mPort,
+        getEndpoint(FileSystemMasterClientRestServiceHandler.GET_STATUS), params, "GET", fileInfo)
+        .run();
 
     Mockito.verify(mFileSystemMaster).getFileInfo(Mockito.<AlluxioURI>any());
   }
 
   @Test
-  public void getStatusInternalTest() throws Exception {
-    Map<String, String> params = new HashMap<>();
-    params.put("fileId", "1");
-
-    FileInfo fileInfo = FileInfoTest.createRandom();
-    Mockito.doReturn(fileInfo).when(mFileSystemMaster).getFileInfo(Mockito.anyLong());
-
-    TestCaseFactory.newMasterTestCase(
-        getEndpoint(FileSystemMasterClientRestServiceHandler.GET_STATUS_INTERNAL), params, "GET",
-        fileInfo, mResource).run();
-
-    Mockito.verify(mFileSystemMaster).getFileInfo(Mockito.anyLong());
-  }
-
-  @Test
-  public void getUfsAddress() throws Exception {
```

Comment:
because the test is ignore, it was not removed when the corresponding endpoint was removed